core-library/src/test/java/org/silverpeas/core/contribution/tracking/ContributionTrackingCreationTest.java

```diff
@@ -71,8 +71,8 @@ class ContributionTrackingCreationTest {
 
   @BeforeEach
   public void setup() {
-    CacheServiceProvider.getThreadCacheService().clearAllCaches();
-    CacheServiceProvider.getRequestCacheService().clearAllCaches();
+    CacheAccessorProvider.getThreadCacheAccessor().getCache().clear();
+    CacheAccessorProvider.getThreadCacheAccessor().getCache().clear();
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** ‏ duplicate of above line